src/main/java/uk/gov/hmcts/sscs/model/ccd/Appeal.java

```diff
@@ -10,15 +10,33 @@
 @Value
 @Builder
 public class Appeal {
+    private String mrnDate;
+    private String mrnLateReason;
+    private String mrnMissingReason;
```

Comment:
mrn details should be grouped into a separate Mrn class

Reply:
Those mrn details are already defined in the ccd def.
If we need to move into seperate class this requires ccd def change. 
It will have an effect if used in the case loader.